stdlib/3/os/__init__.pyi

```diff
@@ -325,8 +321,11 @@ if sys.version_info >= (3, 5):
     def scandir(path: str = ...) -> Iterator[DirEntry[str]]: ...
     @overload
     def scandir(path: bytes) -> Iterator[DirEntry[bytes]]: ...
+@overload
+def stat_float_times(newvalue: bool = ...) -> None: ...
```

Comment:
I don't think this should have a default value; that will make it overlap with the other overload.